adaptor-config.properties.sample

```diff
@@ -54,6 +54,21 @@ documentum.src =
 # are pushed to GSA.
 # documentum.pushLocalGroupsOnly = false
 
+# Custom query to send modified documents. Incremental updates uses this query
+# to find the modified documents and send to GSA. {0} will be replaced by the
+# modified date and {1} will be replaced by the object ID. The select list
+# needs to include r_modify_date_str, r_object_id, i_chronicle_id
+# and object_name.
```

Comment:
It also needs to order by the modify time and object ID (though not strictly by the dm_sysobject attributes). Also, we should point out somewhere, either here or in the doc, that dm_document is the documentTypes property value, and /Repository is the src property value.

What would you think about putting all of this into a wiki page instead, and either leaving this out of adaptor-config.properties entirely, or just referencing the wiki page for detailed requirements? This is not a trivial property to set correctly, so I'm tempted to make it less obvious.